onlyoffice/ui/dev/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Extension Demo</title>
```

Comment:
Replace `Extension Demo` with `OnlyOffice` in this entire file, if the file is needed.